### PR TITLE
refactor: Suppress large printout of Geant4

### PR DESCRIPTION
**Pull request by benjaminhuth**
This PR suppresses the large printout of physics parameters at the beginning of each Sequencer run by default.

### CODE REVIEW

File: Examples/Algorithms/Geant4/src/Geant4Simulation.cpp [hunk 1]
```diff
@@ -84,6 +87,13 @@ ActsExamples::Geant4Simulation::Geant4Simulation(
   G4EventManager::GetEventManager()->GetStackManager()->SetVerboseLevel(
       geantVerboseLevel);
 
+  // Suppress the printing of physics information.
+  if (m_cfg.printPhysicsParameters) {
+    G4HadronicParameters::Instance()->SetVerboseLevel(geantVerboseLevel);
+    G4HadronicProcessStore::Instance()->SetVerbose(geantVerboseLevel);
+    G4EmParameters::Instance()->SetIsPrintedFlag(true);
```
Comment (andiwand) [inline Examples/Algorithms/Geant4/src/Geant4Simulation.cpp:94]: looks like this does not compile. are you using a different G4 version?

**Review by andiwand — APPROVED** [review]
looks good but there seem to be some built issues

File: Examples/Algorithms/Geant4/src/Geant4Simulation.cpp (hunk 1 above)
Comment (benjaminhuth) [reply Examples/Algorithms/Geant4/src/Geant4Simulation.cpp:94]: Probably, I mostly use very recent builds from G4... Hopefully there is some kind of `#ifdef` mechanism in G4 for different versions...

File: Examples/Algorithms/Geant4/src/Geant4Simulation.cpp [hunk 2]
```diff
@@ -84,6 +88,15 @@ ActsExamples::Geant4Simulation::Geant4Simulation(
   G4EventManager::GetEventManager()->GetStackManager()->SetVerboseLevel(
       geantVerboseLevel);
 
+  // Suppress the printing of physics information.
+  if (m_cfg.printPhysicsParameters) {
+    G4HadronicParameters::Instance()->SetVerboseLevel(0);
```
Comment (andiwand) [inline Examples/Algorithms/Geant4/src/Geant4Simulation.cpp:93]: now clang tidy complains that this does not exist :disappointed: 

maybe we can just move everything into the version switch?

Comment (benjaminhuth) [reply Examples/Algorithms/Geant4/src/Geant4Simulation.cpp]: yeah lets do this

**Review by andiwand — APPROVED** [review]

**Pull request merged**